src/hotel_reservations/data_processor.py

```
@@ -1,50 +1,133 @@
-            ]
+        train_set_with_timestamp.write.mode("append").saveAsTable(self.train_table_uc)
+        test_set_with_timestamp.write.mode("append").saveAsTable(self.test_table_uc)
+
+        spark.sql(f"ALTER TABLE {self.train_table_uc} " "SET TBLPROPERTIES (delta.enableChangeDataFeed = true);")
+        spark.sql(f"ALTER TABLE {self.test_table_uc} " "SET TBLPROPERTIES (delta.enableChangeDataFeed = true);")
+
+        train_set_spark = spark.sql(f"select * from {self.train_table_uc}")
+        test_set_spark = spark.sql(f"select * from {self.test_table_uc}")
+
+        return train_set_spark, test_set_spark
+    
```

Ruff

Blank line contains whitespace

Remove whitespace from blank line

(W293)